src/_P007_PCF8591.ino

```diff
@@ -48,6 +45,36 @@ boolean Plugin_007(uint8_t function, struct EventStruct *event, String& string)
       break;
     }
 
+    case PLUGIN_WEBFORM_SHOW_I2C_PARAMS:
+    {
+      String  portNames[4];
+      int     portValues[4];
+      uint8_t unit    = (CONFIG_PORT - 1) / 4;
+      uint8_t port    = CONFIG_PORT - (unit * 4);
+      uint8_t address = 0x48 + unit;
+
+      for (uint8_t x = 0; x < 4; x++) {
+        portValues[x] = x + 1;
+        portNames[x]  = F("A");
```

Comment:
Can't this be initialized with `'A'` ?
char vs flash string does save quite a bit in binary size.